addons/sale_purchase_stock/tests/test_lead_time.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from datetime import timedelta
+from datetime import timedelta, date
```

Comment:
Instead of `datetime.date`, please use `fields.Date`, it ensures more consistent behavior (e.g. datetime.datetime has slightly different ms values compared to fields.Datetime, not sure about date, but may as stick with the fields version)